includes/class-sensei-updates.php

```diff
@@ -88,11 +88,26 @@ public function run_updates() {
 		$this->v4_10_update_install_time();
 		$this->v4_12_create_default_emails();
 		$this->v4_19_2_update_legacy_quiz_data();
+		$this->v4_24_1_update_capabilities();
 
 		// Flush rewrite cache.
 		Sensei()->initiate_rewrite_rules_flush();
 	}
 
+	/**
+	 * Update capabilities.
+	 *
+	 * @since 4.24.1
+	 */
+	private function v4_24_1_update_capabilities() {
```

Comment:
Since we changed capabilities, we need a migration to apply the changes. It only happens on the plugin activation.